solutions/platinum/spoj-kprimesb.mdx

```diff
@@ -0,0 +1,70 @@
+---
```

Comment:
hey uh next time can you not put so much stuff in a single pr lol, makes it kinda hard to review

Reply:
Sure